src/yardstick/store/scan_result.py

```diff
@@ -24,15 +24,15 @@ def _store_root(store_root: str | None = None):
 
 
 def clear(store_root: str | None = None):
-    try:
+    with contextlib.suppress(FileNotFoundError):
```

Comment:
Same comment as above. Why not pass `ignore_errors=True`? https://docs.python.org/3/library/shutil.html#shutil.rmtree